nxdrive/gui/api.py

```diff
@@ -443,6 +444,51 @@ def default_server_url_value(self) -> str:
         """Make daily job better for our developers :)"""
         return getenv("NXDRIVE_TEST_NUXEO_URL", "")
 
+    @pyqtSlot(str, int, result=list)
+    def disk_space_info_to_width(self, uid: str, total_width: int) -> List[float]:
+        engine = self._manager.engines.get(uid)
+        if not engine:
+            return [0, 0, 0]
```

Comment:
And so when returning a tuple, no need for parenthesis:
```suggestion
            return 0, 0, 0
```